api/custom_auth/urls.py

```diff
@@ -26,7 +26,11 @@
         CustomAuthTokenLoginWithMFACode.as_view(),
         name="mfa-authtoken-login-code",
     ),
-    path("logout/", TokenDestroyView.as_view(), name="authtoken-logout"),
+    path(
```

Comment:
I think we need to delete the JWT_SLIDING_COOKIE_KEY cookie here, it looks to just invalidate rather than remove it too.

Reply:
I also noticed as a result, /auth/login/ wouldn't work until I force deleted the cookie. This makes me wonder if the backend needs to ignore the cookie on certain endpoints. This could cause an issue for invite workflows where the user registers / logs in with a cookie set in order to accept an invite.

Reply:
This is done.